tdrs-backend/tdpservice/users/api/utils.py

```diff
@@ -21,7 +21,7 @@
 logger = logging.getLogger(__name__)
 
 now = datetime.datetime.now()
-timeout = now + datetime.timedelta(minutes=settings.SESSION_TIMEOUT)
+timeout = now + datetime.timedelta(minutes=settings.SESSION_COOKIE_AGE)
```

Comment:
Irrelevant to your PR, but could you remove the code for (response_internal)[https://github.com/raft-tech/TANF-app/blob/156d26b04b45aa9e4d8ee5b7c6333a4b4234b9a4/tdrs-backend/tdpservice/users/api/utils.py#L162] function since it is not used anywhere?

Reply:
I am not clear on why we need to set timeout and perform .set_cookie in _response_redirect_ function. Django is supposed to take care of set_cookie automatically (see http.cookies), using simple_cookie class. I think we were doing set_cookie to ensure SESSION_TIMEOUT is in effect but we shouldn't need it anymore, It redirects to `/` on the frontend. Does the frontend need specific cookie settings?

Reply:
i can't say i have the answer to your question, but i think the whole auth module could use a rework.